cloudfoundry-client-lib/src/main/java/org/cloudfoundry/client/lib/util/RequestOrdererV3.java

```diff
@@ -0,0 +1,32 @@
+package org.cloudfoundry.client.lib.util;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface RequestOrdererV3 {
+    
+    String orderBy();
+
+    OrderDirectionV3 direction();
```

Comment:
Shouldn't we have a default direction here: `ASCENDING`. That's what the API says at least.

Reply:
Good point.